xarray/tests/test_backends.py

```diff
@@ -3004,6 +3004,60 @@ def test_open_mfdataset_does_same_as_concat(self, combine, opt, join):
                 ds_expect = xr.concat([ds1, ds2], data_vars=opt, dim="t", join=join)
                 assert_identical(ds, ds_expect)
 
+    @pytest.mark.parametrize(
+        ["combine_attrs", "attrs", "expected", "expect_error"],
+        (
+            pytest.param("drop", [{"a": 1}, {"a": 2}], {}, False, id="drop"),
+            pytest.param(
+                "override", [{"a": 1}, {"a": 2}], {"a": 1}, False, id="override"
+            ),
+            pytest.param(
+                "no_conflicts", [{"a": 1}, {"a": 2}], None, True, id="no_conflicts"
```

Comment:
Hmm... `expected=None` looks like a bug to me. Shouldn't `.attrs` always return a dictionary?

Reply:
this set of parameters has `expect_error=True`, so `expected` is not used at all

Reply:
Haha. Missed that. Thanks!